Tests/SwiftFormatTests/PrettyPrint/CommaTests.swift

```diff
@@ -0,0 +1,145 @@
+import SwiftFormat
+
+final class CommaTests: PrettyPrintTestCase {
```

Comment:
Can you add some tests that cover dictionary literals as well?

Reply:
Note that tests for single-line dictionary setters wrap differently than single-line array setters, that's reflected in the expected test output.